canonical/ubuntu.com
# Add appropriate labels to all form elements

## Done

- I went through [this document](https://github.com/canonical/ubuntu.com/files/10677373/HTML.form.control.has.no.accessible.name.ods) provided through our a11y audit and updated the forms where appropriate. A lot of pages share the same form.
- There is one field in red which is not tackled in this PR and will require upstream changes, these changes can be found in [this issue](https://warthogs.atlassian.net/browse/WD-1894)

## QA

- I advise installing a a11y extension to help with the process, I used [wave](https://wave.webaim.org/) for this PR. This will allow you to check if a page is breaking any a11y rules. You can expect many rules to be broken as this PR only tackles form labels.
- Go to each individual page listed in the document above and check that there are no label/form related rules being broken.
- Go to each page and activate your screen reader, make sure the form groups and labels are being read.

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-1562
